pkg/operator/controller/dns/controller.go

```diff
@@ -54,6 +54,13 @@ const (
 	// operator's namespace that will hold the credentials that the operator
 	// will use to authenticate with the cloud API.
 	cloudCredentialsSecretName = "cloud-credentials"
+
+	// kubeCloudConfigNamespace is the namespace where the kube cloud config ConfigMap is located
+	kubeCloudConfigNamespace = "openshift-config-managed"
```

Comment:
This namespace constant is already available here: https://github.com/openshift/cluster-ingress-operator/blob/master/pkg/operator/controller/names.go#L16

Would you mind using the existing definition in place of a new constant?

Reply:
Yep. I will make that change.